Kitodo/src/main/java/de/sub/goobi/helper/ldap/Ldap.java

```diff
@@ -456,21 +454,19 @@ public boolean changeUserPassword(User inUser, String inOldPassword, String inNe
             env.put(Context.SECURITY_CREDENTIALS, ConfigCore.getParameter("ldap_adminPassword"));
 
             try {
-                DirContext ctx = new InitialDirContext(env);
-
                 /*
                  * Encryption of password and Base64-Encoding
                  */
                 MessageDigest md = MessageDigest.getInstance(ConfigCore.getParameter("ldap_encryption", "SHA"));
                 md.update(inNewPassword.getBytes(StandardCharsets.UTF_8));
-                String digestBase64 = new String(Base64.encodeBase64(md.digest()), StandardCharsets.UTF_8);
+                String digestBaseSixtyfour = new String(Base64.encodeBase64(md.digest()), StandardCharsets.UTF_8);
```

Comment:
There are a cases when warning from checkstyle can be ignored. It is one of them.

Reply:
My work finished by wendnesday cannot be contributed because of checkstyle warnings. So I want to remove a lot of them to be able to contribute to the project again. This rule is made of a regular expression. So if this is not what we want we should discuss changing the regular expression instead.

Reply:
My opinion is that this check style rules should be reviewed and adjusted after a used time (maybe every 2 years). If the current rules does not allow numbers in variable names then we should respect this and change it accordingly.